### DISCUSSION TITLE
Upgrading linux version or transferring to newer system

### DISCUSSION THREAD

**Discussion by farokh-ny** [postfixadmin/postfixadmin#763] [Q&A] [answered]
I've got a couple of systems running Ubuntu 18 LTS, and I've tried upgrading to a newer version, but I have run into issues where the postfix/dovecot system is broken after the upgrade. I'm using postfixadmin, and I'm wondering if there's something I need to do to make this work.

Alternatively, is there a way to move the email data (users, email etc) to a new system?

Thanks!

**Comment by DavidGoodwin** [comment]
Hi - if you were able to supply error message(s) I suspect someone could help.

**Reply by farokh-ny** [reply]
I will make the attempt again (it's a VM so I just snapshot it and run the upgrade) and post the results.
Thanks!

**Reply by farokh-ny** [reply, marked as answer]
I tried it again, and it turned out that there were issues with the mysql and dovecot confs  that prevented them from starting after the upgrade. I fixed that and it appears to be working OK.
